spec/lib/validators/quality_title_validator_spec.rb

```diff
@@ -41,32 +31,34 @@ class Validatable < OpenStruct
     expect(topic).to be_valid
   end
 
-  it "allows anything in a private message" do
```

Comment:
Is this no longer true?

Reply:
I realized there condition was in the struct.

Reply:
Yeah this test wasn't really doing anything useful because the logic for private messages was in the `QualityTitleValidatorSpec::Validatable` class which lived in the tests...